spec/rails_helper.rb

```diff
@@ -12,6 +12,15 @@
 require File.join(File.dirname(__FILE__), 'support/user_sign_in_helper')
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
 
+require 'capybara/poltergeist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.